omopetl/transform.py

```diff
@@ -72,6 +84,23 @@ def transform_normalize_date(self, source_column, target_column, transformation)
         date_format = transformation.get("format", "%Y-%m-%d")
         return pd.to_datetime(self.data[source_column], errors="coerce").dt.strftime(date_format)
 
+    def generate_person_id(subject_id):
+        """
+        
+        To generate a universal unique identifier for each patient
+        
+        """
+        generated_uuid = uuid.uuid4()
+        return farmhash.fingerprint64(str(generated_uuid))
+
+    def coalesce(*args):
+        """
+        
+        Return 0 if the result is NULL
```

Comment:
`Return 0 if the result is NULL` doesn't really describe the behaviour of the function. Instead how about something like: `Return the first non-null value from a list of arguments.`